lib/models/schema.js

```diff
@@ -240,6 +230,7 @@ class Schema extends Base {
    * @returns {Object<string, Schema|string[]>}
    */
   dependencies() {
+    /* eslint-disable security/detect-object-injection */
```

Comment:
why disable, we we cannot just use Map here. Which part linter doesn't like?

Reply:
Linter doesn't like the `key` variable. If you use `Object.keys()` then you have 100% sure, than variable is string and we don't do things like `_json.dependencies[key]()`, so we avoid eval malicious code, so I decided to disable plugin in those lines. I could use utils helper `createMapOfType`, but as you can see, in one case we create Schema Type, in another we write raw value.

Reply:
I don't like to disable rules unless it is a test or there is really nothing else we can do.
values are accessed by a key and with Map it is not the case so it is better, so I don't really get your explanation about `_json.dependencies[key]()`